servicemesh-release-notes/topics/rn-known-issues.adoc

```diff
@@ -37,13 +37,42 @@ oc process -f istio_product_operator_template.yaml | oc delete -f -
 
 https://issues.jboss.org/browse/MAISTRA-5[MAISTRA-5] - `openshift-ansible-istio-installer-job` pod tries to start but with errors.
 
+Graph layout - The layout for the Kiali graph may render differently, depending on your application architecture and the data to display (number of graph nodes and their interactions).  Because it is difficult if not impossible to create a single layout that could render nicely for every situation, Kiali offers a choice of several different layouts.  To choose a different layout you can choose a different Layout Schema from the Graph Settings menu.
```

Comment:
s/may/can
no could
s/different layout/different layout,
Surround the UI elements with *